`make distclean` fails when configured with `--enable-ztex`

I tried `make distclean` and it works for regular configurations. But `make distclean` exits with error after `./configure ... --enable-ztex`:

```
$ ./configure --disable-openmp --disable-opencl --enable-ztex
[...]

$ make distclean
[...]
make[1]: Entering directory '[...]/JohnTheRipper/src/ztex'
make[1]: *** No rule to make target 'distclean'.  Stop.
make[1]: Leaving directory '[...]/JohnTheRipper/src/ztex'
Makefile:1835: recipe for target 'distclean' failed
make: *** [distclean] Error 1

$ echo $?
2
```

john is fresh from the repo: last commit is f389ca4c87e93e55ffb8a38202e11905bd4f29ff.

--- a/src/ztex/Makefile.in
+++ b/src/ztex/Makefile.in
@@ -8,6 +8,7 @@ CC = gcc
 CFLAGS = -c -Wall -O2
 CFLAGS_TEST = -O
 #LD = ld
+RM = /bin/rm -f
 EXTRA_LIBS = -lusb-1.0
 
 SUBDIRS = pkt_comm
@@ -73,3 +74,6 @@ pkt_test: pkt_test.c $(SUBDIRS) $(TEST_OBJS)
 clean:
 	find . -name \*.o -exec rm -f "{}" \;
 	rm -f *.exe simple_test test pkt_test
+
+distclean: clean
+	$(RM) Makefile

